src/Microsoft.Health.Fhir.Azure/Resources.Designer.cs

```diff
@@ -94,5 +94,29 @@ internal class Resources {
                 return ResourceManager.GetString("InvalidStorageUri", resourceCulture);
             }
         }
+
+        internal static string ContainerRegistryNotAuthorized
+        {
+            get
+            {
+                return ResourceManager.GetString("ContainerRegistryNotAuthorized", resourceCulture);
+            }
+        }
+
+        internal static string ContainerRegistryNotConfigured
+        {
+            get
+            {
+                return ResourceManager.GetString("ContainerRegistryNotConfigured", resourceCulture);
+            }
+        }
+
+        internal static string ContainerRegistryNotRegistered
```

Comment:
>ContainerRegistryNotRegistered [](start = 31, length = 30)

nit: I think this file need to be rebuild (or whatever right word to apply custom tool to regenerate it), since .resx file has only two new entities, and here we have three.

Reply:
Agree. Seems regeneration from my side outputs a different format with many comments, so I just removed the extra item manually (the code is the same with the generated one).